src/client/NextApi.Client/NextApiClient.cs

```diff
@@ -181,7 +181,7 @@ protected virtual HubConnection InitializeClientSignalR()
                     await Task.Delay(ReconnectDelayMs);
                     await connection.StartAsync();
                 };
-            connection.On("NextApiEvent", new[] {typeof(NextApiEventMessage)}, ProcessNextApiEvent);
+            connection.On("NextApiEvent", new[] { typeof(NextApiEventMessage) }, ProcessNextApiEvent);
```

Comment:
Not really critical, but it is better to leave it be, or add these whitespaces everywhere to have same code-style.